js/jquery.indiciaFootableReport.js

```diff
@@ -95,6 +95,20 @@
           }
         });
       }
+
```

Comment:
I think the Elasticsearch dataGrid only uses Footable to provide responsive cols, plus a client-side sort in the scenario where aggregations don't allow sort. So there are cases where Footable is not loaded but the dataGrid can be sorted server-side. Therefore would it be better to place this code inside the dataGrid's JS file?

Reply:
I've changed it John in line with your comment.